mm2src/coins/lp_coins.rs

```diff
@@ -2100,6 +2104,30 @@ impl MmCoinEnum {
     }
 }
 
+impl MmPlatformCoin for MmCoinEnum {
+    fn on_token_deactivated(&self, ticker: &str) -> Result<(), String> {
```

Comment:
Why not just make `fn on_token_deactivated` part of MmCoin trait?

Reply:
I think having `MmPlatformCoin` is not bad too incase of further operations on Platform Coins specifically. @sergeyboyko0791 wdyt?

Reply:
Multiple coin types don't need to implement `on_token_deactivated`, so it breaks the [Interface segregation principle
](https://en.wikipedia.org/wiki/Interface_segregation_principle). It was my suggestion to add a new `MmPlatformCoin` trait for such purposes.

But as I can see, `MmPlatformCoin` is implemented for every coin within `MmCoinEnum`.
Also when I suggested adding `MmPlatformCoin` I forgot that there is `EthCoin` that can be both `ETH` and `ERC20`, so this principle is broken already.

@borngraced could you please move `on_token_deactivated` method to `MmCoin` trait?